pymeasure/instruments/keithley/keithley2700.py

```diff
@@ -248,8 +248,8 @@ def channels_from_rows_columns(self, rows, columns, slot=None):
             columns = new_columns
```

Comment:
This file is not part of your this pull request and it is actually part of the master branch.
Again, something wrong with rebase ?